lib/two_factor_authentication/version.rb

```diff
@@ -1,3 +1,3 @@
 module TwoFactorAuthentication
-  VERSION = "1.1.5".freeze
+  VERSION = "1.1.6".freeze
 end
```

Comment:
The version number update is usually left to the maintainer. Not every PR might merit a version update. Some people like to combine several contributions into a version update.